.circleci/deploy.sh

```diff
@@ -8,10 +8,11 @@
 set -e -x
 
 function delete_requests() {
+    git pull
```

Comment:
hmmm... In some rare cases, this could cause an error in the jobs that are initializing just then..